spock/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.jboss.weld</groupId>
+      <artifactId>weld-junit-parent</artifactId>
+      <version>4.0.0-SNAPSHOT</version>
+   </parent>
+
+   <artifactId>weld-spock</artifactId>
+
+   <properties>
+      <spotbugs.excludeFilterFile>${project.basedir}/spotbugs-exclude.xml</spotbugs.excludeFilterFile>
+   </properties>
+
+   <dependencies>
+
+      <dependency>
+         <groupId>org.jboss.weld.se</groupId>
+         <artifactId>weld-se-core</artifactId>
+      </dependency>
+
+      <dependency>
+         <groupId>org.jboss.weld</groupId>
+         <artifactId>weld-junit-common</artifactId>
+      </dependency>
+
+      <!-- Spock dependencies -->
+      <dependency>
+         <groupId>org.spockframework</groupId>
+         <artifactId>spock-core</artifactId>
+         <scope>provided</scope>
```

Comment:
It would make sense, unfortunately it is not really possible.
There is a Groovy 2.5 variant and a Groovy 3.0 variant and soon a Groovy 4.0 variant.
Which one you need depends on the Groovy version you use to write the tests.
But including this module only makes sense if Spock is available anyway, so it isn't too much of a problem I'd say.